build/build/src/engine/context.rs

```diff
@@ -338,55 +338,30 @@ impl RunContext {
         // we don't want to call this in environments like GH-hosted runners.
 
         // === Build project-manager distribution and native image ===
-        debug!("Bulding project-manager distribution and Native Image");
-        if crate::ci::big_memory_machine() {
-            let mut tasks = vec![];
-
-            if self.config.build_engine_package() {
-                tasks.push("buildEngineDistribution");
-                tasks.push("engine-runner/assembly");
-            }
-            if self.config.build_native_runner {
-                tasks.push("engine-runner/buildNativeImage");
-            }
-
-            if self.config.build_project_manager_package() {
-                tasks.push("buildProjectManagerDistribution");
-            }
-
-            if self.config.build_launcher_package() {
-                tasks.push("buildLauncherDistribution");
-            }
+        let mut tasks = vec![];
+        if self.config.build_engine_package() {
+            tasks.push("engine-runner/assembly");
+            tasks.push("buildEngineDistribution");
+        }
```

Comment:
`buildEngineDistribution` is a command required for many other commands (for example, dry-run benchmarks and Enso tests). I believe that, at least on Linux, this command is usually run as the very first command. We should ensure that this is run also on other architectures. If not, we may bump into weird dependency issues. Please ensure that `self.config.build_engine_package` is set on MacOS and Windows as well.